Formula/racer.rb

```diff
@@ -0,0 +1,33 @@
+class Racer < Formula
+  desc "Rust Code Completion utility"
+  homepage "https://github.com/phildawes/racer"
+
+  url "https://github.com/phildawes/racer/archive/v1.1.0.tar.gz"
+  sha256 "f969e66d5119f544347e9f9424e83d739eef0c75811fa1a5c77e58df621e066d"
+  head "https://github.com/phildawes/racer.git"
+
+  depends_on "rust" => :build
+
+  resource "rust_source" do
+    url Formula["rust"].stable.url
```

Comment:
I'm not thrilled with making this dynamic since it means a given (formula version, formula revision, bottle revision) tuple can actually mean more than one thing, and the additional information about which specific version of the sources was used wouldn't make it into the bottle or receipt.